sources/packages/backend/apps/api/src/route-controllers/student-appeal/student-appeal.institutions.controller.ts

```diff
@@ -0,0 +1,49 @@
+import { Controller, Param, Get, ParseIntPipe } from "@nestjs/common";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { 
```

Comment:
Formatter is showing error because of additional space here for me. Are other devs seeing it?

![image](https://github.com/bcgov/SIMS/assets/54600590/82a99d1c-f03c-4a75-a5fe-0430e1401f74)

Reply:
Yes, same for me @dheepak-aot. When I save the file it is fixed.
@ann-aot is VS Code formatting on save?

Reply:
ya @andrewsignori-aot. When i was working on this file, VS code was slow and i forced shutdown, probably at that time I have missed it. I have fixed it now